wolox-books/src/app/screens/login/login.component.ts

```diff
@@ -11,38 +7,9 @@ import {inputLogin} from './constants.js';
 })
 export class LoginComponent implements OnInit {
 
-  registerForm: FormGroup;
-  post: any;
-  messageAlert: string = 'Campo obligatorio';
-  inputData: string = inputLogin;
-
-  constructor( private fb: FormBuilder, private registerUserService: RegisterUserService ) { }
-
-  ngOnInit() {
-    this.buildForm();
-  }
-
-  private buildForm(){
-    this.registerForm = this.fb.group({
-      'firstName': [null, Validators.compose([Validators.required, Validators.minLength(3)])],
-      'lastName': [null, Validators.compose([Validators.required, Validators.minLength(3)])],
-      'email': [null, Validators.compose([Validators.required, Validators.email])],
-      'password': [null, Validators.compose([Validators.required, Validators.maxLength(20), Validators.minLength(5)])],
-      'passwordConfirmation': [null, Validators.compose([Validators.required, Validators.maxLength(20), Validators.minLength(5)])]
-    });
-  }
-
-  addPost() {
-
-    const data = {
-          password_confirmation: this.registerForm.value.passwordConfirmation,
-          first_name: this.registerForm.value.firstName,
-          last_name: this.registerForm.value.lastName,
-          locale: "en",
-          ...this.registerForm.value
-    }
+  constructor() { }
 
-    this.registerUserService.userRegister(data).subscribe(responsePost => console.log('Succesfull', responsePost) );
+  ngOnInit(): void {
```

Comment:
remove ngOnInit if it's not used, remove the implementation in the class definition too

Reply:
I do not delete this coz the next card I need this component and class